src/Network/Email/AbstractTransport.php

```diff
@@ -1,3 +1,16 @@
 <?php
-// @deprecated Backward compatibility with 2.x, 3.0.x
-class_alias('Cake\Mailer\AbstractTransport', 'Cake\Network\Email\AbstractTransport');
+// @codingStandardsIgnoreFile
+namespace Cake\Network\Email;
+
+class_alias(\Cake\Mailer\AbstractTransport::class, AbstractTransport::class);
+
+if (class_exists(AbstractTransport::class)) {
+    return;
+}
```

Comment:
@chinpei215 @ADmad @markstory what if I add a unit test that instantiate all the aliases classes, and the all the old classes. If that works without an error it should be safe.